xinference/api/restful_api.py

```diff
@@ -1292,6 +1293,33 @@ async def create_embedding(self, request: Request) -> Response:
         payload = await request.json()
         body = CreateEmbeddingRequest.parse_obj(payload)
         model_uid = body.model
+
+        # 检查 body.input 是否是一个二维整数列表
```

Comment:
Could you write the comments in English?

Reply:
Below are the same for a couple of comments.

Reply:
OK